services/src/main/java/org/fao/geonet/api/eealayoutapi/EEALayoutManager.java

```diff
@@ -0,0 +1,228 @@
+        Log.info(API.LOG_MODULE_NAME, "Updating EEA template layout from remote...");
+
+        String tempHEAD = loadHeadFromAPI();
+        String tempSCRIPTS = loadScriptFromAPI();
+        String tempHEADER = loadHeaderFormEEAapi();
+        String tempFOOTER = loadFooterFormEEAapi();
+
+        HEAD = tempHEAD;
+        SCRIPTS = tempSCRIPTS;
+        HEADER = tempHEADER;
+        FOOTER = tempFOOTER;
+
+    }
+
+    public static String getHEAD() {
+        if (HEAD == null) {
+            try {
+                Log.info(API.LOG_MODULE_NAME, "Updating EEA head layout from remote...");
+                HEAD = loadHeadFromAPI();
+            } catch (Exception e) {
+                Log.error(API.LOG_MODULE_NAME, "Error in updating EEA head from remote API", e);
+                return "";
+            }
+        }
+
+        return HEAD;
+    }
+
+    public static String getSCRIPTS() {
+        if (SCRIPTS == null) {
+            try {
+                Log.info(API.LOG_MODULE_NAME, "Updating EEA scripts layout from remote...");
+                SCRIPTS = loadScriptFromAPI();
+            } catch (Exception e) {
+                Log.error(API.LOG_MODULE_NAME, "Error in updating EEA scripts from remote API", e);
+                return "";
+            }
+        }
+
+        return SCRIPTS;
+    }
+
+    public static String getHEADER() {
+        if (HEADER == null) {
+            try {
+                Log.info(API.LOG_MODULE_NAME, "Updating EEA header layout from remote...");
+                HEADER = loadHeaderFormEEAapi();
+            } catch (Exception e) {
+                Log.error(API.LOG_MODULE_NAME, "Error in updating EEA header from remote API", e);
+                return "";
+            }
+        }
+
+        return HEADER;
+    }
+
+    public static String getFOOTER() {
+        if (FOOTER == null) {
+            try {
+                Log.info(API.LOG_MODULE_NAME, "Updating EEA footer layout from remote...");
+                FOOTER = loadFooterFormEEAapi();
+            } catch (Exception e) {
+                Log.error(API.LOG_MODULE_NAME, "Error in updating EEA footer from remote API", e);
+                return "";
+            }
+        }
+
+        return FOOTER;
+    }
+
+    private static String loadHeadFromAPI() throws Exception {
+        URL url;
+        InputStream is = null;
+        BufferedReader inputReader = null;
+        try {
+            url = new URL(readProperty("eea.api.template.head"));
+            is = url.openStream(); // throws an IOException
+            inputReader = new BufferedReader(new InputStreamReader(is));
+
+            StringBuilder sb = new StringBuilder();
+            String inline;
+
+            sb.append("<head>");
+            while ((inline = inputReader.readLine()) != null) {
+
+                sb.append(inline);
+            }
+            sb.append("<script src=\"../../srv/api/eealayoutupdate/script\" /></head>");
```

Comment:
Why ? @PascalLike

Reply:
Because the javascript contains characters not compatible with the XSLT engine. In this way, we can insert the javascript code externally from the HTML and avoid to break the XSLT engine

Reply:
Ok but we get 
![image](https://user-images.githubusercontent.com/1701393/43948151-50d39882-9c8a-11e8-9540-2805773cb2c4.png)
for this script URL

Reply:
It's better to put this part directly in the HTML page I think. Doing it